api/src/main/resources/workbench-api.yaml

```diff
@@ -7230,6 +7230,10 @@ components:
           type: boolean
           default: false
           description: If true, allow for user's initial credits to expire.
+        enableUnlinkBillingForInitialCredits:
```

Comment:
do you need this in the api?

Reply:
This is used in InitialCreditsService.java in the stopInitialCreditSpendInWorkspace function. Does that usage make sense?

Reply:
I see what you mean now. I removed it from the yaml and ConfigResponse.